index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=
+    , initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="landingPage/style.css">
```

Comment:
change the css path to be like that ... To avoid path read error when we deploy the project into Github pages

```html
<link rel="stylesheet" href="./landingPage/style.css">
```